src/parser/test/ParserTest.cpp

```diff
@@ -403,7 +410,7 @@ TEST(Parser, InsertEdge) {
     {
         GQLParser parser;
         std::string query = "INSERT EDGE NO OVERWRITE transfer(amount, time) "
-                            "VALUES 12345->54321@1537408527:(3.75, 1537408527)";
+                            "VALUES 12345->-54321@1537408527:(3.75, 1537408527)";
         auto result = parser.parse(query);
         ASSERT_TRUE(result.ok()) << result.status();
     }
```

Comment:
try -1->-2

Reply:
What's your point?

Reply:
a negtive vid to another negtive vid. recommend to cover this simple situation.